api-design-guidelines/index.md

```diff
@@ -933,7 +933,7 @@ x.move(**from:** x, **to:** y)
 
 ## Special Instructions
 
-* **Label closure parameters and tuple members** where they
+* **Label tuple members and name closure parameters** where they
```

Comment:
Is this supposed to say `named closure parameters` rather than `name closure parameters`?

Reply:
It's grammatical either way but the meaning changes fairly significantly.

Reply:
What's there is what I intended. Closure parameters should be "named", tuple members should be "labeled". We seem to be using "label" where something is exposed to clients, and closure parameters won't have that effect.

Reply:
Okay, that sounds accurate; nevermind! Looking at the diff overemphasizes that you swapped out two noun phrases with a noun phrase + verb phrase, which ends up being fine but set off my suspicion that it could have been an unintentional near miss.

Reply:
Thanks for proofreading in the first place. :-)